README.md

```diff
@@ -54,9 +54,24 @@ For critical bug fixes on production
 - Install EF Core CLI tools (https://docs.microsoft.com/en-us/ef/core/cli/dotnet)
 - Node v14+ (https://nodejs.org/en/)
 - If you're using Rider then you will need to install the ".net core user secrets" plugin
+- Download and configure Minio for Windows (https://min.io/download#/windows) (see below)
 
 In HerPortal run `npm install`
 
+#### Minio
+
+The portal site lists files hosted in an S3 bucket. For local development we need a fake S3 bucket to connect to.
+To use [Minio](https://min.io/) to provide a local S3 bucket follow these steps:
+1. [Download minio](https://min.io/download#/windows)
+2. Put the executable somewhere on your machine (e.g. c:\Program Files\Minio)
+3. Decide on a folder for Minio to store its data in (e.g. c:\data\minio)
```

Comment:
If you want a more self-contained solution, it looks like you can also run it in a docker container - e.g. we could set up a `docker-compose.yml` that runs the app, database and local S3 simultaneously to avoid needing to install any external dependencies except docker itself.

Reply:
Perhaps one for the support team to consider

Reply:
Again, trying to avoid docker to save licences :)

Reply:
Ah, fair enough.

I don't really know the alternatives (podman?) well enough to know if they're worth pursuing, I suspect it's unlikely to pay off for this project unless you already have experience.